docs/rules/default-case.md

```diff
@@ -78,13 +78,20 @@ switch (a) {
     // no default
 }
 
+switch (a) {
+    case 1:
+        /* code */
+        break;
+
+    // No Default
+}
 ```
 
 ## Options
 
 This rule accepts a single options argument:
 
-* Set the `commentPattern` option to a regular expression string to change the default `^no default$` comment test pattern
+* Set the `commentPattern` option to a regular expression string to change the default `^no default$/i` comment test pattern
```

Comment:
If we want to add a slash to show the case-insensitive flag, I think it'd be better to have a leading slash as well: `/^no default$/i`